lib/plugins/backup.rb

```diff
@@ -0,0 +1,124 @@
+require 'rubygems/package'
+require 'zlib'
+require_relative '../moonshot/creds_helper'
+
+module Moonshot::Plugins # rubocop:disable Documentation
+  # Monkeypatching TarWriter for our
```

Comment:
Truth be told, `add` isnt a method in TarWriter, it just makes the implementation way easier if I patch it to the class.